test/hotspot/jtreg/runtime/valhalla/inlinetypes/TestFieldTypeMismatch.java

```diff
@@ -28,27 +28,22 @@
 /**
  * @test
  * @library /test/lib
- * @compile TestFieldTypeMismatchHelper.jasm
- * @compile TestFieldTypeMismatch.java
+ * @build org.openjdk.asmtools.* org.openjdk.asmtools.jasm.*
+ * @run driver org.openjdk.asmtools.JtregDriver jasm -strict TestFieldTypeMismatchClasses.jasm
```

Comment:
Naming convention change: suggest putting jasm classes in a file named `[Test name]Classes.jasm`. No need for any classes declared there to actually use that name, although they should have names that won't clash with other tests in this directory.

Reply:
Good idea